### PR TITLE
Added icons.

### PR DESCRIPTION
**Pull request by nowarkdev**

### CODE REVIEW

**Comment by bentidy** [comment]
Can we remove all the unneeded stuff in the svg? sketch/ids/comments etc

File: package.json [hunk 1]
```diff
@@ -29,7 +29,7 @@
     "npm": "3.3.12"
   },
   "dependencies": {
-    "nowtv-grid": "git://github.com/sky-uk/grid.git"
+    "nowtv-grid": "git://github.com/sky-uk/grid.git#1ce875932712c2e5cb04ed910e2b9de8bdff0dec"
```
Comment (peter-mouland) [inline package.json]: whats this commit hash in here, i don't think the styleguide needs it

**Comment by peter-mouland** [comment]
cool, maybe putting them through SVG min would be good

**Comment by nowarkdev** [comment]
@peter-mouland Shouldn't svg-min be part of webpack? or is that what you meant by your comment to add it to webpack.

**Comment by nowarkdev** [comment]
@peter-mouland Created [an issue](https://github.com/sky-uk/nowtv-styleguide/issues/26) for svg-min. This PR was just to move the icons from react-toolkit into the style guide.

**Comment by bentidy** [comment]
:+1: 

@rogerfranklin2 @ldabiralai

**Comment by peter-mouland** [comment]
cool :+1:

**Pull request merged**